lib/components/layout.tsx

```diff
@@ -84,7 +84,7 @@ export const Layout: React.FC<React.PropsWithChildren<Props>> = React.memo(({ ch
 
         .main {
           display: flex;
-          max-width: calc(100% - 100px);
+          max-width: calc(100% - 200px);
```

Comment:
Why change the `max-width` here?

Reply:
left side bar's width is 200px, `100% - 100px` cause `main` cross the `layout` border

![image](https://user-images.githubusercontent.com/1267447/90211049-3b197580-de22-11ea-991c-b3c1557d9712.png)